modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -450,7 +450,7 @@
             ;; hence, assert it was not called anymore here
             (is (= 0 @call-count) "convert-dataset-id-to-filters! should not have been called any more times"))
           ;; now, so we need to manually update the temp DB again here, to force the "old" structure
-          (let [updated? (db/update! Database db-id :details {:dataset-id "my-dataset"})]
+          (let [updated? (t2/update! Database db-id {:details {:dataset-id "my-dataset"}})]
             (is updated?)
```

Comment:
if the update fails will t2 return something falsey? I'm worried the `(is updated?)` isn't testing anything any more

Reply:
yep it should be wrapped with a `pos?`. thanks for finding it.

Reply:
btw: `t2/update!` returns # of affected rows, whereas `db/update!` returns a boolean.

I had a note about this in the PR's description.